test/gleam_stdlib_test_helpers_ffi.mjs

```diff
@@ -0,0 +1,8 @@
+export function uint8array(list) {
```

Comment:
Let's stick to the existing naming convention for this ffi file.

Reply:
That would be?
`gleam_stdlib_ffi.mjs`??